src/codeu/chat/client/simplegui2/ChatMenu.java

```diff
@@ -0,0 +1,200 @@
+package codeu.chat.client.simplegui2;
+
+import codeu.chat.client.ClientContext;
+import javafx.event.ActionEvent;
+import javafx.event.EventHandler;
+import javafx.geometry.Insets;
+import javafx.geometry.Pos;
+import javafx.scene.Scene;
+import javafx.scene.control.Button;
+import javafx.scene.control.Label;
+import javafx.scene.control.TextArea;
+import javafx.scene.layout.BorderPane;
+import javafx.scene.layout.HBox;
+import javafx.scene.layout.VBox;
+import javafx.stage.Stage;
+
+/**
+ * Created by nora on 4/6/17.
+ */
+public class ChatMenu {
+
+private final ClientContext clientContext;
+
+    private BorderPane mainWindow;
+    private Scene scene;
+    private Stage stage;
+
+    //ChatMenu constructor
+    public ChatMenu(ClientContext clientContext){
+        this.clientContext=clientContext;
+
+
+        initialize();
+        stage = new Stage();
+        scene= new Scene(mainWindow);
+        stage.setScene(scene);
+
+    }
+
+    //Public method to call in order to display the main chat stage
+    public void display(){
+        stage.show();
+    }
+
+    //initializes the different panels within the grid layout
+    private void initialize(){
+
+        mainWindow=new BorderPane();
+        mainWindow.setMinHeight(700);
+        mainWindow.setMinWidth(700);
+
+        //organize main Pane
+        mainWindow.setTop(topMenuBar());
+        mainWindow.setLeft(sideBar());
+        mainWindow.setBottom(textBar());
+        mainWindow.setCenter(centralTextBox());
+
+    }
+
+    private HBox topMenuBar(){
+
+        //create menu at top
+        HBox topbar= new HBox(10);
+        topbar.setMaxHeight(100);
+        topbar.setPadding(new Insets(15, 12, 15, 12));
+        topbar.setStyle("-fx-background-color: #336699;");
+
+        //add buttons for different options
+        final Button signOut = new Button("Sign Out");
+        signOut.setPrefSize(100, 20);
```

Comment:
these button sizes should be stored in a shared variable.  Alternatively you could write a function that generates a button of your default size and other properties.